third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -318,6 +318,7 @@ addNewScriptHandler(async (scriptId, sourceURL, relativeSourceMapURL) => {
     targetContentHash: `sha256:${scriptHash}`,
     targetURLHash: sourceURL ? makeAPIHash(sourceURL) : undefined,
     targetMapURLHash: makeAPIHash(sourceMapURL),
+    timestamp: Date.now(),
```

Comment:
Your PR has no link to a Linear issue. Can you provide a link (and file a Linear issue if you don't have one) and explain what all of this is supposed to do?

I'm very reluctant to just take on any runtime changes that are not properly documented.

Reply:
My thought is this:

1. I want to thank you for explaining the PR to begin with. Before that, I did not have any context, now at least I know why you have made this change.
2. This change is more intrusive than meets the eye. It adds a system call in our own injected JS code. It should be Ok, but its important to have an eye out for this when working on the runtime.
3. This line should already add the timestamp: https://github.com/replayio/backend/blob/a2311478c1f4e8919acb45618dca177c24a0d408/src/cpp/driver/RecordingEvents.cpp#L223

If it did not get added, I'd like to know why before trying to paper over this.

Reply:
Can you provide more data for the "some of the logs were missing a timestamp" bug?

Reply:
(And this is why I would want a Linear issue 😜 )

Reply:
Aha... The reason why the timestamp was not added is because this is an entirely different implementation of `AddRecordingEvent`, defined [here](https://github.com/replayio/chromium/blob/00aa96c87e8286f42c9cbdfdb1a1187c45301992/third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc#L1211).